superset-frontend/stylesheets/superset.less

```diff
@@ -600,3 +600,55 @@ input[type='radio']:checked:after {
 hr {
   border-top: 1px solid @gray-light;
 }
+
+.modal-fullscreen {
+  & > .modal-lg {
+    height: calc(100% - 60px);
+    & > .modal-content {
+      height: 100%;
+      display: flex;
+      flex-direction: column;
+      & > .modal-body {
+        display: flex;
+        flex-direction: column;
+        flex: 1;
+      }
+    }
+  }
+}
+
+.flex-1 {
+  flex: 1;
+}
+
+.sticky-table {
+  & > div {
+    & > div:nth-child(2) {
+      & > div:first-child {
+        & > table {
+          margin-bottom: 0;
+          th {
+            border-bottom-width: 1px;
+          }
+        }
+      }
+      & > div:nth-child(2) {
+        &::-webkit-scrollbar {
+          -webkit-appearance: none;
+          &:vertical {
+            width: 7px;
```

Comment:
Not sure if this 7px measurement (and the one below it) need to be exactly 7px, but if it's just _about that big_ then if this were moved to Emotion I would do `width: ${({ theme }) => theme.gridUnit * 2}px` to make it themable
```
            width: 7px;
```